### DISCUSSION TITLE
question: additional camera view in cockpit

### DISCUSSION THREAD

**Discussion by zcaalock** [Mogli12/KeyboardSteer#750] [Ideas]
is this possible in fs19/22? The goal is to make easier to see the tool while playing from FPS perspective

In this example camera is showing rear view but it would be nice to set it wherever player wants

http://www.farming2015mods.com/wp-content/uploads/2016/05/REAR-VIEW-CAMERA-V1-FS-2015.jpg

**Comment by Mogli12** [comment]
There were several mods up to FS17 with rear view. But this did not work anymore in FS19.
